src/wled/const.py

```diff
@@ -0,0 +1,56 @@
+"""Asynchronous Python client for WLED."""
+
+from enum import IntEnum, IntFlag
+
+from awesomeversion import AwesomeVersion
+
+MIN_REQUIRED_VERSION = AwesomeVersion("0.14.0")
+
+
+class LightCapability(IntFlag):
```

Comment:
@frenck 
The values seem incorrect (https://kno.wled.ge/interfaces/json-api/#light-capabilities), can you check?

Reply:
This was already there and not part of this PR. If you find an issue, please create an issue.

Reply:
You are right, I'm going to do that, ty